test/functional/cmdLineTests/CDSAdaptorTest/playlist.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2020, 2020 IBM Corp. and others
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../TKG/playlist.xsd">
+	<test>
+		<testCaseName>CDSAdaptorTest</testCaseName>
+		<variations>
+			<variation>NoOptions</variation>
+		</variations>
+		<command>cp $(Q)$(TEST_RESROOT)$(D)cdsadaptortest.jar$(Q) .; \
+		cp $(Q)$(LIB_DIR)$(D)org.eclipse.osgi_3.16.100.v20200904-1304.jar$(Q) .; \
```

Comment:
- Instead of coping the jar into current dir, can we just directly point to the jar under $(LIB_DIR) in cdsadaptortest.xml?
- Is it possible to do the same for jars in cdsadaptortest.jar? It seems that we jar it in build.xml and unjar it in playlist.xml. It would be nice if we can avoid this by not jarring it at all.

Reply:
Optimized the process as suggested, used ${LIB_DIR} for jar, and run test without copying into output folder through cdsadaptortest.jar.